src/issue/screens/new-issue.screen.js

```diff
@@ -106,71 +140,91 @@ class NewIssue extends Component {
-                })
-              }
-              onChangeText={text => this.setState({ issueTitle: text })}
-              placeholderTextColor={colors.grey}
-              value={issueTitle}
-              valueHeight={issueTitleHeight}
-            />
-          </SectionList>
-
-          <SectionList title={t('Issue Comment', locale)}>
-            <StyledTextInput
-              underlineColorAndroid={'transparent'}
-              placeholder={t('Write a comment for your issue here', locale)}
-              multiline
-              onChangeText={text => this.setState({ issueComment: text })}
-              onContentSizeChange={event =>
-                this.setState({
-                  issueCommentHeight: event.nativeEvent.contentSize.height,
-                })
-              }
-              placeholderTextColor={colors.grey}
-              value={issueComment}
-              valueHeight={issueCommentHeight}
-            />
-          </SectionList>
-
-          <SectionList>
-            <SubmitView>
-              <SubmitListItem
-                title={t('Submit', locale)}
+        <Animated.View style={{ flex: 1, paddingBottom: keyboardHeight }}>
+          <ScrollView
+            ref={ref => {
+              this.scrollViewRef = ref;
+            }}
+            onContentSizeChange={() => this.scrollViewRef.scrollToEnd()}
+          >
+            {repository.full_name && (
+              <StyledListItem
+                title={repository.full_name}
+                leftIcon={{
+                  name: 'repo',
+                  size: 17,
+                  color: colors.grey,
+                  type: 'octicon',
+                }}
                 hideChevron
-                underlayColor={colors.greyLight}
-                onPress={() => this.submitNewIssue()}
               />
-            </SubmitView>
-          </SectionList>
-        </ScrollView>
-      </ViewContainer>
+            )}
+            <SectionList title={t('Issue Title', locale)}>
+              <StyledTextInput
+                underlineColorAndroid={'transparent'}
+                placeholder={t('Write a title for your issue here', locale)}
+                blurOnSubmit
+                multiline
+                onContentSizeChange={event =>
+                  this.setState({
+                    issueTitleHeight: event.nativeEvent.contentSize.height,
+                  })
+                }
+                onChangeText={text => this.setState({ issueTitle: text })}
+                placeholderTextColor={colors.grey}
+                value={issueTitle}
+                valueHeight={issueTitleHeight}
+                onFocus={() => {
+                  this.shouldScroll = false;
+                }} // donot scroll to bottom when this input is focused
+                onBlur={() => {
+                  this.shouldScroll = true;
+                }} // when this input is not focused, scrollview should scroll to bottom when keyboard show up
```

Comment:
`when keyboard show up` -> `when keyboard shows up` or `when keyboard is showing up`? In fact, I am not a native English speaker. :P

Reply:
Sorry didn't pay much attention to grammar. Fully focused on solving the problem :P. Will correct it. 👍